### PR TITLE
Update test_loss.py

### PR DESCRIPTION
**Pull request by DOnlyRaven**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py [hunk 1]
```diff
@@ -491,3 +492,15 @@ def test_paddle_nll_loss(
         ignore_index=ignore_index,
         reduction=reduction,
     )
+def softmax_cross_entropy(logits, labels):
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py:495]: Hey! Thanks for contributing! Please link the issue you're working on to the PR, and the frontend function should be added to the functional file, this test file should only consist of the test functions, formatted like the existing ones. The outermost logits, labels and print statements should not be included as well, thanks!

**Comment by juliagsy** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
